[CIVIS] Better explain billing exclusions for CI Visiblity (#26976)
DataDog/documentation: content/en/account_management/billing/ci_visibility.md

A committer is an active Git contributor, identified by their Git author email address. For billing purposes, a committer is included if they make at least three commits in a given month.

> ## Charges for bot or actions in the GitHub UI

Reviewer: ```suggestion
## Charges for bot commits and GitHub UI actions
```

Author: ```suggestion
## Charges for commits made by bots or actions performed in the GitHub UI
```

> Datadog does not charge for bot or actions made in the GitHub UI. These types of committers are excluded from billing calculations.

Reviewer: ```suggestion
Datadog does not charge for commits made by bots or actions performed in the GitHub UI. These activities are excluded from billing calculations.
```

Author: Actions are not billed, this is the commit that results from that action that could be billed. So I would say:
```suggestion
Datadog does not charge for bot or commits resulting from actions made in the GitHub UI. These types of commits are excluded from billing calculations.
```

Author: Same for the title above

Reviewer: Noted, that makes sense to me! Let me know once you apply your second revision and I can approve this. 🚀
